v1/firebase/adapter.js

```diff
@@ -40,7 +40,18 @@ function apiGetTrack(trackId) {
 
 function apiGetChannel(channelId) {
 	return apiGet(`channels/${channelId}`).then(snapshot => {
-		return serializeChannel(snapshot.val(), channelId);
+		var channel = serializeChannel(snapshot.val(), channelId);
+
+		// Replace `images` with a single `thumbnail` URL.
+		var images = Object.keys(channel.images);
+		var lastImage = images[images.length - 1];
```

Comment:
Please keep in mind that property order is not guaranteed in JS. It will probably work, but its kind of yucky to rely on that.